src/main/java/frc/robot/Robot.java

```diff
@@ -79,7 +97,28 @@ public void autonomousPeriodic() {
 
     @Override
     public void testInit() {
+        ShooterPID shooterPID = new ShooterPID(shooterSystem,5000);
```

Comment:
why did you add all these commands?

Reply:
these commands were there before, I added mine at the end. I'll delete them